insert_data_to_vector_db_weaviate/main.bal

```diff
@@ -0,0 +1,62 @@
+import ballerina/io;
+import ballerinax/googleapis.sheets;
+import ballerinax/openai.embeddings;
+import ballerinax/weaviate;
+
+configurable string openAIToken = ?;
+configurable string weaviateToken = ?;
+configurable string weaviateURL = ?;
+
+configurable string sheetAccessToken = ?;
+configurable string sheetId = ?;
+configurable string sheetName = ?;
+
+const RANGE = "A2:C630";
+const NO_OF_COLUMNS = 3;
+string className = "QuestionAnswerStore";
```

Comment:
Convert to a const?

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/60/commits/b266f95fb9a512010774915db533652910a42502.